prometheus-exporter/templates/aws-credentials.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This is really specific to an exporter that has to connect to the AWS API. I think you should either do a specific Helm chart, or create a generic one without this